tests/math_builtin_api/modules/sycl_types.py

```diff
@@ -1226,33 +1196,29 @@ def create_types():
 
 
     # t_char_n = argtype("charn", "NULL", "NULL", 0, ["sycl::char2","sycl::char3","sycl::char4","sycl::char8","sycl::char16"])
```

Comment:
This comment is outdated now. I suggest removing such comments.

Reply:
Arguably the aliases are still valid given the aliases map to [fixed-size integer vectors](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_aliases) despite using the names of fundamental types. That said, I don't know why the comments were there in the first place, given they seem to be valid definitions that have been commented out. That considered, would you still like me to remove them?

Reply:
Do you mean that this file is used to generate variants for something else other than builtins?

Reply:
I'm asking this because technically, your patch is removing "arguably valid aliases".

Reply:
I don't believe so, no. This should all just be for builtins but it seems like we elected to write out the types of the aliases instead of using the aliases defined in the spec. At least that is my guess as to why these were commented out. Note though that these aliases changed from 1.2.1 to 2020 to be more in line with how the corresponding vector types in OpenCL are fixed-size.

Reply:
Not sure if this is important, but we plan to deprecate these "vec" aliases.  See the comments at the bottom of KhronosGroup/SYCL-Docs#353.

Reply:
Okay. @steffenlarsen, up to you. I'm waiting for another approval. I'll merge with the comment if you decide to leave it.

Reply:
I think I agree with removing them, but I would prefer to do it in a separate patch as there are also non-integer aliases that this likely affects. I will open a separate patch for removing the comments.

Reply:
Correction. It seems to only be the integer ones, so I will include it in this patch.